src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/Internal/TimeSeriesFormCallout.tsx

```diff
@@ -151,8 +160,20 @@ const TimeSeriesFormCallout: React.FC<IProp> = ({
                     propertyExpression={{
                         expression: seriesToEdit?.expression
                     }}
+                    description={
+                        seriesToEdit?.propertyType &&
```

Comment:
if we make the propertyType required as I suggested in my previous comment, lets remove this  `seriesToEdit?.propertyType` check from here.

Reply:
true, why not just alwyas have a value and then check if it's string since that's the only time we want to show this?